config/tests/samples/create/harness.go

```diff
@@ -673,7 +673,7 @@ func MaybeSkip(t *testing.T, name string, resources []*unstructured.Unstructured
 			case schema.GroupKind{Group: "compute.cnrm.cloud.google.com", Kind: "ComputeNodeGroup"}:
 			case schema.GroupKind{Group: "compute.cnrm.cloud.google.com", Kind: "ComputeNodeTemplate"}:
 			case schema.GroupKind{Group: "compute.cnrm.cloud.google.com", Kind: "ComputeManagedSSLCertificate"}:
-			//case schema.GroupKind{Group: "compute.cnrm.cloud.google.com", Kind: "ComputeServiceAttachment"}:
```

Comment:
is this intended?

Reply:
Yes, we started noticing this deletion issue in a forwarding rule test with service attachment dependency, and it blocked presubmits. So, we disabled this mock service. #1069

I added it back here because the issue has been addressed in this PR, and the presubmits all passed now.